README.md

```diff
@@ -75,3 +75,13 @@ Topic:topic2    PartitionCount:5        ReplicationFactor:1     Configs:retentio
 ```
 
 Any changes to the `test-topics.yml` file will be applied to the existing topics at each subsequent run.
+
+## Docker
+
+The Docker image is available from Docker Hub.
```

Comment:
Yup good point I think splitting it is a good idea. I moved it all down this morning as wanted to have `Usage` before the example of running the image, splitting it sounds like a good plan and then maybe have the Docker example as part of the `Demo` section.